docs/getting-started/install/linux/index.md

```diff
@@ -12,7 +12,7 @@ Regardless of your distribution, you will need the Mono Project GPG signing key,
 Debian, Ubuntu, and derivatives
```

Comment:
Yes, just follow https://github.com/mono/website#setting-up-a-local-copy-of-the-website.

Newer Jekyll also supports `--incremental` so the whole site isn't rebuilt all the time which takes a while.